v3/plugins/che-incubator/typescript/1.35.1/meta.yaml

```diff
@@ -17,3 +17,5 @@ spec:
     memoryLimit: '512Mi'
   extensions:
   - https://download.jboss.org/jbosstools/vscode/3rdparty/ms-code.typescript/che-typescript-language-1.35.1.vsix
+  - https://github.com/theia-ide/vscode-node-debug/releases/download/v1.35.3/node-debug-1.35.3.vsix
```

Comment:
no-debug is deprecated and only needed for Node < 8. 

https://github.com/eclipse/che/issues/14927 
https://github.com/eclipse/che-plugin-registry/pull/302

We removed it from Che. Please don't add it back.